DataFormats/Detectors/MUON/MCH/include/DataFormatsMCH/Digit.h

```diff
@@ -40,10 +41,10 @@ class Digit
   int32_t getTime() const { return mTFtime; }
 
   void setNofSamples(uint16_t n);
-  uint16_t nofSamples() const { return (mNofSamples & 0x7FFF); }
+  uint16_t nofSamples() const;
 
   void setSaturated(bool sat);
-  bool isSaturated() const { return ((mNofSamples & 0x8000) > 0); }
+  bool isSaturated() const;
```

Comment:
The idea is to avoid to expose (too many) implementation details in the header file. And our [coding guidelines](https://rawgit.com/AliceO2Group/CodingGuidelines/master/coding_guidelines.html?showone=Inline_Functions#Inline_Functions) don't object to that ("Small functions, like accessors and mutators _may_ be placed" ).
I'll change the nofSamples to getNofSamples to comply with the naming conventions (just have to cross-check the repercutions for QC for instance), as I missed [that one](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Function_Names#Function_Names) indeed.

Reply:
Well, it is a tiny but unnecessary overhead...